controller/workitem.go

```diff
@@ -71,6 +72,33 @@ func NewNotifyingWorkitemController(service *goa.Service, db application.DB, not
 		config:       config}
 }
 
+// authorizeWorkitemTypeEditor returns true if the modifier is allowed to change
+// workitem type else it returns false.
+// Only space owner and workitem creator are allowed to change workitem type
+func (c *WorkitemController) authorizeWorkitemTypeEditor(ctx context.Context, db application.DB, spaceID uuid.UUID, creatorID string, editorID string) (bool, error) {
```

Comment:
There is no need for `db application.DB` parameter in this function. It can be safely removed.